mappings/net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen.mapping

```diff
@@ -1,6 +1,20 @@
 CLASS net/minecraft/class_415 net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen
 	FIELD field_2438 confirmButton Lnet/minecraft/class_4185;
 	FIELD field_2441 biomeSelectionList Lnet/minecraft/class_415$class_4190;
+	FIELD field_24562 parent Lnet/minecraft/class_437;
+	FIELD field_24563 onClickDoneButton Ljava/util/function/Consumer;
```

Comment:
Just `onDone`
```suggestion
	FIELD field_24563 onDone Ljava/util/function/Consumer;
```

Reply:
I really don't know why I didn't add this with the other changes suggested on the review, sorry